set correct base
set proper path for log and temp dirs
remove link to internal wrapper - doesn't work
set link to tanukiwrapper

Index: installer/resources/i2prouter
--- installer/resources/i2prouter.orig
+++ installer/resources/i2prouter
@@ -28,7 +28,7 @@
 # Note that (percent)INSTALL_PATH, (percent)USER_HOME,  and (percent)SYSTEM_java_io_tmpdir
 # should have been replaced by the izpack installer.
 # If you did not run the installer, replace them with the appropriate paths.
-I2P="%INSTALL_PATH"
+I2P="${LOCALBASE}/share/i2p"
 if [ "`uname -s`" = "Darwin" ]; then
    if [ -d "%USER_HOME/Library/Application Support" ]; then
        I2P_CONFIG_DIR="%USER_HOME/Library/Application Support/i2p"
@@ -36,9 +36,9 @@ if [ "`uname -s`" = "Darwin" ]; then
        I2P_CONFIG_DIR="%USER_HOME/.i2p"
    fi
 else
-    I2P_CONFIG_DIR="%USER_HOME/.i2p"
+    I2P_CONFIG_DIR="${DB_DIR}"
 fi
-I2PTEMP="%SYSTEM_java_io_tmpdir"
+I2PTEMP="${DB_DIR}"
 # PORTABLE installation:
 # Use the following instead.
 #I2PTEMP="%INSTALL_PATH"
@@ -69,7 +69,7 @@ fi
 #RUN_AS_USER=
 
 # Wrapper
-WRAPPER_CMD="$I2P/i2psvc"
+WRAPPER_CMD="${LOCALBASE}/sbin/tanukiwrapper"
 WRAPPER_CONF="$I2P/wrapper.config"
 
 # Priority at which to run the wrapper.  See "man nice" for valid priorities.
